source/MapDetailPanel.cpp

```diff
@@ -448,102 +507,89 @@ void MapDetailPanel::DrawKey()
 // details, trade prices, and details about the selected object.
 void MapDetailPanel::DrawInfo()
 {
-	const Color &faint = *GameData::Colors().Get("faint");
 	const Color &dim = *GameData::Colors().Get("dim");
 	const Color &medium = *GameData::Colors().Get("medium");
 
-	Point uiPoint(Screen::Left() + 100., Screen::Top() + 45.);
+	const Color &back = *GameData::Colors().Get("map side panel background");
+
+	static const Interface *planetCardInterface = GameData::Interfaces().Get("map planet card");
```

Comment:
Same thing with `static` in this file as well.

Reply:
I already removed all the inapropriate statics based on what you said
I should just remove all of them and whatever the performance cost? Yeah ik this is the map interface panel it doesn't matter